apps/web/src/pages/activities/ActivitiesPage.tsx

```diff
@@ -139,63 +131,92 @@ export function ActivitiesPage() {
-                />
-              )}
-              control={control}
-              name="transactionId"
-            />
-          </div>
-          <div style={{ minWidth: '250px' }}>
-            <Controller
-              render={({ field, fieldState }) => (
-                <Input
-                  {...field}
-                  error={fieldState.error?.message}
-                  label="Search"
-                  placeholder="Search by subscriberId or email"
-                  value={field.value || ''}
-                  onChange={(e) => {
-                    field.onChange(e);
-                    debouncedSearchChange(e.target.value);
-                  }}
-                  data-test-id="search-filter"
-                />
-              )}
-              control={control}
-              name="search"
-            />
-          </div>
-          {isDirty && (
-            <Button variant="outline" size="md" mt={30} onClick={onClearClick} data-test-id="clear-filters">
-              Clear
-            </Button>
-          )}
+            <div style={{ minWidth: '250px' }}>
+              <Controller
+                render={({ field, fieldState }) => (
+                  <Input
+                    {...field}
+                    error={fieldState.error?.message}
+                    label="Transaction ID"
+                    placeholder="Search by transaction id"
+                    value={field.value || ''}
+                    data-test-id="transactionId-filter"
+                  />
+                )}
+                control={control}
+                name="transactionId"
+              />
+            </div>
+
+            <div style={{ minWidth: '250px' }}>
+              <Controller
+                render={({ field, fieldState }) => (
+                  <Input
+                    {...field}
+                    error={fieldState.error?.message}
+                    label="E-mail"
+                    placeholder="Search by subscriber email"
+                    value={field.value || ''}
+                    data-test-id="email-filter"
+                  />
+                )}
+                control={control}
+                name="email"
+              />
+            </div>
+            <div style={{ minWidth: '250px' }}>
+              <Controller
+                render={({ field, fieldState }) => (
+                  <Input
+                    {...field}
+                    error={fieldState.error?.message}
+                    label="Subscriber ID"
+                    placeholder="Search by subscriberId"
+                    value={field.value || ''}
+                    data-test-id="subscriberId-filter"
+                  />
+                )}
+                control={control}
+                name="subscriberId"
+              />
+            </div>
+            <div style={{ paddingTop: 10, marginLeft: 'auto' }}>
+              <Button
+                variant="subtle"
```

Comment:
```suggestion
                variant="outline"
```
<img width="348" alt="Screen Shot 2023-07-16 at 13 12 02" src="https://github.com/novuhq/novu/assets/7778680/6343fa21-3d5f-42af-abe4-b76c5c3f5e79">